traffic_ops/testing/api/v4/serverservercapability_test.go

```diff
@@ -222,6 +223,43 @@ func GetTestServerServerCapabilities(t *testing.T) {
 	}
 }
 
+func UpdateTestServerServerCapabilities(t *testing.T) {
+	var header http.Header
+
+	// Get server capability name and edit it to a new name
+	resp, _, err := TOSession.GetServerCapabilitiesWithHdr(header)
+	if err != nil {
```

Comment:
Same comment about `len(resp)` here.

Reply:
Same comment as above, in GetServerCapabilityWithHdr(), the check is already present and hence I didn't add the check.

Reply:
Done